src/components/sectionsWheelPicker/SectionsWheelPicker.driver.tsx

```diff
@@ -1,21 +1,35 @@
-import _ from 'lodash';
+import React from 'react';
 import {useComponentDriver, ComponentProps} from '../../testkit/new/Component.driver';
 import {WheelPickerDriver} from '../WheelPicker/WheelPicker.driver';
 import {SectionsWheelPickerProps} from './index';
 
-export const SectionsWheelPickerDriver = (props: ComponentProps) => {
-  const driver = useComponentDriver<SectionsWheelPickerProps>(props);
-  const sections = driver.getProps().children as SectionsWheelPickerProps;
-  const sectionsDrivers = _.map(sections, (_, index) => {
-    const sectionTestID = `${props.testID}.${index}`;
```

Comment:
add `    console.log('sectionWheel: ', section?.props.testID);` here and get the passed testID

Reply:
I think the only thing we need is to pass the section's testID since the component already sets it if the user didn;t override it

Reply:
This would work. I thought about the case if we ever add some component along side the sections so we will return that other thing as a Wheel driver. Like if we add some label or something.